packages/cli/README.md

```diff
@@ -0,0 +1 @@
+# cli
```

Comment:
Can we call the package something a bit more helpful eg `deployment-cli`? 😁

Reply:
was thinking reapit-cli